src/util.c

```diff
@@ -17,3 +17,36 @@ long parse_int(char *str) {
 
     return parsed;
 }
+
+unsigned long parse_buttons_numbers(char *str,
+ignore_buttons_t *ignore_buttons) {
```

Comment:
The formatting here is weird with no indentation at all. Can we just move this onto the previous line?